### DISCUSSION TITLE
can buildah build overwrite existing manifest entry?

### DISCUSSION THREAD

**Discussion by lazywhite** [containers/buildah#5035] [Q&A] [answered]
I use following steps to build my image
```
# 1. create manifest
buildah manifest create demo:v1
# 2. add image to manifest
buildah build --manifest demo:v1 --platform linux/arm64 .
```
sometime I need to execute step2 again to  rebuild image after some code changes, 
but it won't overwrite the existing entry which has the same platform.
how can I do it?

**Comment by flouthoc** [comment, marked as answer]
@lazywhite You can do it like this, if you want to overwrite with a new image.

*) `buildah build --platform linux/arm64 .`
*) `buildah manifest add demo:v1 containers-storage:<image-id-or-name>`

Overall `buildah build` should also do overwrite automatically if not then its a bug, you can open an issue upstream so it can be fixed.

Thanks.
